utils/utils_test.go

```diff
@@ -107,3 +107,24 @@ func TestDeleteTempFileFail(t *testing.T) {
 	_, err := DeleteTempFile("TestFile.yaml")
 	assert.EqualError(t, err, errString)
 }
+
+func TestRemoveDuplicateEntries(t *testing.T) {
+	dupArr := []string{"test", "test", "test"}
+	result := RemoveDuplicateEntries(dupArr)
+
+	if len(result) != 1 {
+		log.Fatal("Test 1: Failed to delete duplicate array index")
+	}
+
+	dupArr = []string{"", "test", "test"}
+	result = RemoveDuplicateEntries(dupArr)
+	if len(result) != 1 {
+		log.Fatal("Test 2: Failed to delete duplicate array index")
+	}
+
+	dupArr = []string{"", "", ""}
```

Comment:
Nit: should `RemoveDuplicateEntries(["", "", ""])` really return `[]`, rather than `[""]`?

Reply:
It does return []. If the length of an array = 0 then there are no elements. The return value is [] and not [""] as this would then be 1.